liwords-ui/src/utils/cwgame/common.ts

```diff
@@ -28,6 +28,13 @@ export const isBlank = (letter: string): boolean => {
   return false;
 };
 
+export const isDesignatedBlank = (letter: string): boolean => {
+  if (letter.toLowerCase() === letter && letter.toUpperCase() !== letter) {
```

Comment:
probably just `return letter.toLowerCase() === letter && letter.toUpperCase() !== letter`

Reply:
I fixed the one above as well